src/models.jl

```diff
@@ -85,25 +86,35 @@ end
 
     kwargs are passed to the regular `Model` constructor.
 """
-ChannelModel(; bcs=ChannelModelBoundaryConditions(), kwargs...) = 
+ChannelModel(; bcs=ChannelModelBCs(), kwargs...) = 
     Model(; bcs=bcs, kwargs...)
           
+#
+# Model initialization utilities
+#
```

Comment:
I like the section headers but the beefier versions from https://github.com/jrevels/YASGuide might be even nicer:
```julia
#####
##### Model initialization utilities
#####
```

Reply:
Yes, I'm happy to change to these. Mind if we save this work for a future PR? We should comb through the code and change all of them at once.

Reply:
Sounds like a good idea. There are other files that could do with some section headers.